src/frameserver/main.cpp

```diff
@@ -241,6 +243,11 @@ int main(int argc, char *argv[]) {
             server = std::make_shared<oat::TestFrame>(sink, file_path, frames_per_second);
             break;
         }
+        case 'e':
+        {
+            server = std::make_shared<oat::PGUSBCam>(sink, index, frames_per_second);
+            break;
```

Comment:
We should move toward `server = std::make_shared<oat::PGCam<USB>>(sink, index, frames_per_second);` and `server = std::make_shared<oat::PGCam<GigE>>(sink, index, frames_per_second);`